tcl/excel.tcl

```diff
@@ -204,7 +282,7 @@ proc qc::excel_file_create {args} {
     ::try {
         log Debug $script_filename
         exec_proxy -timeout $timeout perl $script_filename
-        file delete $script_filename
+        #file delete $script_filename
```

Comment:
Is this meant to still be commented out?

Reply:
No, fixed.